src/main/java/org/mskcc/kickoff/printer/MappingFilePrinter.java

```diff
@@ -197,9 +225,12 @@ public void print(Request request) {
 
                     String[] paths = sampleFqPath.split("\n");
 
-                    // Find out if this is single end or paired end by looking inside the directory for a R2_001.fastq.gz
+                    // Find out if this is single end or paired end by looking inside the directory for a
+                    // R2_001.fastq.gz
                     for (String path : paths) {
-                        if ((sampleId.contains("POOLEDNORMAL") && (sampleId.contains("FFPE") || sampleId.contains("FROZEN"))) && !path.matches("(.*)IGO_" + request.getBaitVersion().toUpperCase() + "_[ATCG](.*)")) {
+                        if ((sampleId.contains("POOLEDNORMAL") && (sampleId.contains("FFPE") || sampleId.contains
```

Comment:
There is also MOUSEPOOLEDNORMAL now. It doesn't really effect anything at this point, but be aware this enumeration isn't complete.

Reply:
mousepoolednormal should be treated the same as other poolednormals? or differently? if the same it will fall into the same category with sampleId.contains("POOLEDNORMAL")